addons/l10n_ar/models/account_move.py

```diff
@@ -167,20 +177,16 @@ def _onchange_partner_journal(self):
                 raise RedirectWarning(msg, action.id, _('Go to Journals'))
 
     def _post(self, soft=True):
-        ar_invoices = self.filtered(lambda x: x.company_id.country_id.code == "AR" and x.l10n_latam_use_documents)
+        posted = super()._post(soft)
```

Comment:
Changing completely the order of operations? Seems like this should change the behavior (I didn't read it more than that)
The position of the call to `super` is usually very important.